shared/profile/edit-avatar/container.js

```diff
@@ -36,17 +44,33 @@ const mapDispatchToProps = (dispatch, ownProps) => ({
     dispatch(ProfileGen.createUploadAvatar({crop, filename})),
 })
 
-const mergeProps = (stateProps, dispatchProps, ownProps) => ({
-  createdTeam: stateProps.createdTeam,
-  image: stateProps.image,
-  onClose: dispatchProps.onClose,
-  onSave: (filename: string, crop?: RPCTypes.ImageCropRect) =>
-    stateProps.teamname
-      ? dispatchProps.onSaveTeamAvatar(filename, stateProps.teamname, stateProps.sendChatNotification, crop)
-      : dispatchProps.onSaveUserAvatar(filename, crop),
-  sendChatNotification: stateProps.sendChatNotification,
-  teamname: stateProps.teamname,
-})
+const networkErrorCodes = [
```

Comment:
we should make this a constant somewhere, config or engine or something. maybe util/errors

Reply:
I tried adding it to `util/errors` but it blew up all the tests, I'm guessing because of the circular import. I'll try again in a separate PR.